<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="HP-Elitebook" failures="1" tests="5" name="D:/JOB FIND 10 days study 28-6/RestAssured" time="13.707" errors="0" timestamp="2024-08-30T13:42:54 IST">
  <testcase classname="Github_API_Automation" name="NoTokenProvided" time="7.782"/>
  <testcase classname="Github_API_Automation" name="InvalidTokenProvided" time="0.527"/>
  <testcase classname="Github_API_Automation" name="ForbiddcenTokenProvided" time="0.443">
    <failure type="java.lang.AssertionError" message="response status code is unexpected as401 expected [403] but found [401]">
      <![CDATA[java.lang.AssertionError: response status code is unexpected as401 expected [403] but found [401]
at Github_API_Automation.ForbiddcenTokenProvided(Github_API_Automation.java:58)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- ForbiddcenTokenProvided -->
  <testcase classname="Github_API_Automation" name="GetUserValidUserToken" time="0.453"/>
  <testcase classname="Github_API_Automation" name="UpdateUserBioWithValidUserToken" time="0.638"/>
</testsuite> <!-- D:/JOB FIND 10 days study 28-6/RestAssured -->
